### ISSUE TITLE
Add scroll to upcoming GP functionality

### ISSUE THREAD

**Issue by moritzjansen** [sportstimes/f1#1838]
Hi there,

I would like to contribute to the F1Calendar by making it possible to automatically scroll to the upcoming GP using the [URI Fragment API](https://developer.mozilla.org/en-US/docs/Web/URI/Fragment).

By simply adding an `id` like `"upcoming"` to the appropriate html element, we could then navigate directly to the upcoming GP with f1calendar.com/#upcoming. This URL could then be bookmarked for users who wish to enable this behaviour by default. 

I would like to know what you think of this idea. If this feature sounds reasonable, I would like to make my first OSS contribution (hype!). I've already scanned the code and found the relevant file and prop needed to make this change. 

Let me know what you think!
Cheers, Mo

**Comment by ay8s** [comment]
@moritzjansen We do have an option to hide previous races which would mean the next race is at the top and open. Wondering if this would solve this or if you think having the #upcoming/#next option would still be useful? 

Either way it should be relatively easy to add an id to the next races row. We already have an isNextRace boolean. 👍

**Comment by moritzjansen** [comment]
Oh nice, I missed that option. That solves my usecase perfectly, thank you!

I played around with the # to see if my approach would even work at all, but it doesn't seem to anyway... E.g. the link 
[https://f1calendar.com/#us-grand-prix](https://f1calendar.com/#us-grand-prix) does not scroll to the intended position, although there is an element with the id `us-grand-prix` at the appropriate position. My initial thought was client side rendering was the issue but thats not the case, any ideas? I stopped digging around when I saw your comment but I'm still curious. 

Thanks for your quick reply and for maintaing this very convenient website, much appreciated 👍
